server/zanata-frontend/src/app/editor/index.html

```diff
@@ -15,7 +15,13 @@
   <meta name="author" content="Red Hat">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="cleartype" content="on">
-
+  <link rel="preconnect" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic" crossorigin />
```

Comment:
I think preconnect is meant to take a prefix only (url plus hostname), not the entire URL.

Maybe we should **preload** the Google Fonts CSS, and **preconnect** https://fonts.gstatic.com/ (which serves the various actual fonts, as requested by the CSS).

    <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>

Reply:
Aha got it, will make this and other changes tomorrow morning.